DZ/DZ/ViewControllerFour.swift

```diff
@@ -0,0 +1,238 @@
+//
+//  ViewControllerFour.swift
+//  lession7 dz
+//
+//  Created by Сережа on 23.05.2023.
+//
+
+import UIKit
+
+class ViewControllerFour: UIViewController {
```

Comment:
```suggestion
class ViewControllerFour: UIViewController {
```
название вью контроллер 4 не говорит о том что на этом экране